tests/app/confirmation_page/test_confirmation_page.py

```diff
@@ -15,7 +15,7 @@ def test_get_next_location_confirmation(self):
         answer_store = AnswerStore()
         answer_store.add(answer)
 
-        navigator = Navigator(load_schema_file("0_rogue_one.json"), answer_store)
+        navigator = Navigator(load_schema_file("0_rogue_one.json"), {}, answer_store)
```

Comment:
but nowhere do we use the navigator without an AnswerStore so why support it? It just means more logic to support/test